forest/src/cli/sync_cmd.rs

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::stringify_rpc_err;
+use actor::EPOCH_DURATION_SECONDS;
+use chain_sync::get_naive_time_now;
+use chrono::naive::NaiveDateTime;
+use chrono::prelude::*;
+use cid::Cid;
+use jsonrpc_v2::Error as JsonRpcError;
+use jsonrpsee::raw::RawClient;
+use jsonrpsee::transport::http::HttpTransportClient as HTC;
+use rpc_client::{check_bad, head, mark_bad, new_client, status, submit_block};
+use std::thread;
+use std::time::Duration;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub enum SyncCommand {
+    #[structopt(
+        name = "mark-bad",
+        about = "Mark the given block as bad, will prevent syncing to a chain that contains it"
+    )]
+    MarkBad {
+        #[structopt(short, long, help = "Block Cid given as string argument")]
+        block_cid: String,
+    },
+
+    #[structopt(
+        name = "check-bad",
+        about = "Check if the given block was marked bad, and for what reason"
+    )]
+    CheckBad {
+        #[structopt(short, long, help = "Block Cid given as string argument")]
+        block_cid: String,
```

Comment:
this seems like it should be a positional arg as well (only one and would remove the need for -b or --block-cid flag)